### PR TITLE
fix(django_admin): Disable django_admin on prod

### PR DESCRIPTION
**Pull request by klboke**
Disable django_admin on prod ，fix #23742 , https://github.com/getsentry/self-hosted/issues/1039

cc @BYK 

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Comment by klboke** [comment]
fix https://github.com/getsentry/self-hosted/issues/1922  cc @hubertdeng123

**Review by BYK — CHANGES_REQUESTED** [review]
Lol, was that it? 😅 

Can we have a smoke test somewhere so we know for sure it is fixed and will stay that way?

**Comment by hubertdeng123** [comment]
> Lol, was that it? 😅

Seems to work, just tested this locally. Thanks @klboke

**Review by hubertdeng123 — APPROVED** [review]
> Can we have a smoke test somewhere so we know for sure it is fixed and will stay that way?

I think this is a nice thing to have but don't think we should block on this.

**Comment by chadwhitacre** [comment]
Let's get a review from the backend team as well.

**Review by markstory — APPROVED** [review]

**Review by dashed — APPROVED** [review]

**Pull request merged**
